crates/runtime/tests/aws_sdk/spicepods/aws_sdk_verify.yaml

```diff
@@ -14,6 +14,7 @@ datasets:
   - from: s3://spiceai-seoul-glue/parquet_taxi_trips_2023/
     name: private_taxi
     params:
+      s3_auth: iam_role
```

Comment:
```suggestion
```

This should work without specifying an explicit `s3_auth` - which is consistent with all of our other AWS components